### ISSUE TITLE
GUI not updating property's values correctly when listening to another property's changes 

### ISSUE THREAD

**Issue by ElonOlsson** [OpenSpace/OpenSpace#3046]
When one property is listening to another propertys changes, the results doesn't seem to update in the gui, but it is setting the correct values.
Pressing f5 to update gui makes it show the correct values.

To reproduce, try any renderable and pick two properties, say _lineWidth and _color.
in the onChange function, make it change color when changing linewidth.
_lineWidth.onChange([this]() {
   _color = glm::vec3(1.f, 0.f, 0.f);
}
The line should be rendered red, but the gui showing the previous color.

**Comment by alexanderbock** [comment]
Mhh.. yea, also can't reproduce in `master`.  Maybe we fixed it along the way?

**Comment by alexanderbock** [comment]
Closing it as it is not an issue

https://github.com/OpenSpace/OpenSpace/assets/4921724/7f78f32a-6b90-48a0-94f6-636bc83303b1

**Comment by alexanderbock** [comment]
To reproduce, add:

```cpp
_lineWidth.onChange([this](){
    glm::vec2 r = _radii;
    r.y = _lineWidth;
    _radii = r;
});
```
to the `RenderableRadialGrid` constructor.

When loading OpenSpace and using the assets/examples/grids.asset, changing the linewidth of the radial grid changes the radii, but the property value is not changed until either F5 or closing the Scene menu and reopening.

**Comment by ylvaselling** [comment]
Fixed [in c9a356ff5a89867cc74d338384955f6e170669d6 on OpenSpace ](https://github.com/OpenSpace/OpenSpace-WebGuiFrontend/tree/c9a356ff5a89867cc74d338384955f6e170669d6)
